.github/Csr.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+
+# kubi private key 
+openssl ecparam -genkey -name secp521r1 -noout -out /tmp/ecdsa-key.pem
+
+# kubi public key
```

Comment:
I probably would generate these keys staticly for CI, and use always the same cert, to minimize surprises...